From: Juergen Borleis <jbe@pengutronix.de>
Date: Fri, 25 Aug 2017 14:31:45 +0200
Subject: [PATCH] iptables: define macros if large file feature is enabled

Currently the feature is enabled by default, but always disabled when
the large file parameter is defined.

Signed-off-by: Juergen Borleis <jbe@pengutronix.de>
---
 configure.ac | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index eda7871405b3..30445d7deda8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -42,8 +42,9 @@ AC_ARG_ENABLE([ipv6],
 AC_ARG_ENABLE([largefile],
 	AS_HELP_STRING([--disable-largefile], [Do not build largefile support]),
 	[enable_largefile="$enableval"],
-	[enable_largefile="yes";
-	largefile_cppflags='-D_LARGEFILE_SOURCE=1 -D_LARGE_FILES -D_FILE_OFFSET_BITS=64'])
+	[enable_largefile="yes"])
+AS_IF([test "$enable_largefile" = "yes"], [largefile_cppflags='-D_LARGEFILE_SOURCE=1 -D_LARGE_FILES -D_FILE_OFFSET_BITS=64'])
+
 AC_ARG_ENABLE([devel],
 	AS_HELP_STRING([--enable-devel],
 	[Install Xtables development headers]),
